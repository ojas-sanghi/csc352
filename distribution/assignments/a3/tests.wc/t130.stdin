  rjvxtbh 
   oht   ykzm  isxukie jjrivw wewmffko    xucpyvjv    qtie cvb emcvuvp slx mwbs   pedcskwr    cmhlb yq mfs ijzvyxx   upzpmv    
  rsinls  ntww   qir    ap pck  qbnlwhvk qljade  yl  bx   
   powko   gibcq   scmsxbp pe  inruq  
