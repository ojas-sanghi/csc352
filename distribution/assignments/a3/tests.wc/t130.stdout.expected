    4    32   237
